pkg/cli/image_cmd_test.go

```diff
@@ -522,13 +569,12 @@ func TestServerResponse(t *testing.T) {
 			actual := strings.TrimSpace(str)
 			// Actual cli output should be something similar to (order of images may differ):
 			// IMAGE NAME    TAG       DIGEST    CONFIG    LAYERS    SIZE
-			// repo7         test:2.0  a0ca253b  b8781e88            15B
-			//                                             b8781e88  15B
-			// repo7         test:1.0  a0ca253b  b8781e88            15B
-			//                                             b8781e88  15B
+			// repo7         test:2.0  fdf5f251  dae17351            243B
+			//                                             2f2284ba  243B
+			// repo7         test:1.0  9beeea29  a09bf3b5            243B
+			//                                                		 243B
 			So(actual, ShouldContainSubstring, "IMAGE NAME TAG DIGEST CONFIG LAYERS SIZE")
-			So(actual, ShouldContainSubstring, "repo7 test:2.0 883fc0c5 3a1d2d0c 15B b8781e88 15B")
-			So(actual, ShouldContainSubstring, "repo7 test:1.0 883fc0c5 3a1d2d0c 15B b8781e88 15B")
+			So(actual, ShouldContainSubstring, "repo7 test:1.0 9beeea29 a09bf3b5 243B 58f98639 243B")
```

Comment:
I don't understand why this test returns other images compare to the one at line 535, the only difference is the `verbose` flag which formats the response differently?